src/lib/TextArea/TextArea.svelte

```
@@ -0,0 +1,223 @@
+    testId,
+    classes = '',
+    footer,
+    onInput,
+    onChange,
+    onFocus,
+    onBlur,
+    onKeyDown,
+    onPaste
+  }: TextAreaProperties = $props();
+
+  let textareaEl: HTMLTextAreaElement | undefined = $state();
```

ESLint

[error] undefined type is not allowed. Use null instead.

(no-restricted-syntax)